packages/services/test/integration/notification-scanners/SubscriptionReminder.js

```diff
@@ -0,0 +1,145 @@
+import chai from 'chai'
+import sinon from 'sinon'
+import sinonChai from 'sinon-chai'
+const { expect } = chai
+chai.use(sinonChai)
+
+import { userDbCleanup, userNotificationTypeDbCleanup } from '../../helpers/dbCleanup'
+import userNotificationTypeByModel from '../../helpers/userNotificationTypeByModel'
+import { tryRunScanner } from '../../../src/workers/notification-scanner'
+import { trySendNotification } from '../../../src/workers/notification-sender'
+import { User, UserEmail, UserNotification } from '@aragonone/court-backend-server/build/models/objection'
+
+const { env: { CLIENT_URL } } = process
+const notificationTypeModel = 'SubscriptionReminder'
+const TEST_ADDR = '0xfc3771B19123F1f0237C737e92645BA6d628e2cB'.toLowerCase()
+const TEST_EMAIL = 'subscription@reminder.test'
+const MINUTES = 60 * 1000
+const HOURS = 60 * MINUTES
+const DAYS = 24 * HOURS
+
+
+describe('SubscriptionReminder notifications', () => {
+
+  after(async () => {
+    await userDbCleanup(TEST_ADDR, TEST_EMAIL)
+    await userNotificationTypeDbCleanup(notificationTypeModel)
+  })
+
+  let logger = {}
+  beforeEach(async () => {
+    logger = {
+      success: sinon.fake(),
+      warn: sinon.fake(),
+    }
+  })
+
+  it('should not create a notification for a user with verified address (opted out)', async () => {
+    await userDbCleanup(TEST_ADDR, TEST_EMAIL)
```

Comment:
It may be useful to create a nested mocha context and use `afterEach()` to do the DB clean up here.

If all the other tests don't rely on carried over state (I think requiring a test with carried over state is usually a red flag in tests though), then maybe we can change the `after()` to be an `afterEach()` :)

Reply:
Yes in this case I'm using stateful tests. That's why I'm calling this an integration test because I depend on the database. 
I've added a PR #199 to make this more explicit